src/domain/enrolment/enrolmentForm/ValidationSchema.tsx

```diff
@@ -14,16 +14,16 @@ export default Yup.object().shape({
       return isSameResponsiblePerson
         ? schema
         : schema.shape({
-            name: Yup.string().required(
-              VALIDATION_MESSAGE_KEYS.STRING_REQUIRED
-            ),
-            phoneNumber: Yup.string().required(
-              VALIDATION_MESSAGE_KEYS.STRING_REQUIRED
-            ),
-            emailAddress: Yup.string()
-              .required(VALIDATION_MESSAGE_KEYS.STRING_REQUIRED)
-              .email(VALIDATION_MESSAGE_KEYS.EMAIL),
-          });
+          name: Yup.string().required(
+            VALIDATION_MESSAGE_KEYS.STRING_REQUIRED
```

Comment:
Didn't git commit prehook works here?

Reply:
No, the pre-hook did not fix these. I had some linter problems, because I had 2 different linters changing the file differently (in vscode). That issue should be fixed now, so this issue will be gone on my next commit.

Reply:
OK that is strange. I just tried locally and pre hook did fix these automatically 🤔

Reply:
Because I had those issues, that 1 file may have been committed with --no-verify.

Reply:
oh ok, that explains